lib/clover_invert.cu

```diff
@@ -17,7 +17,8 @@ namespace quda {
       compute_tr_log(compute_tr_log)
     {
       strcat(aux, compute_tr_log ? ",trlog=true" : "trlog=false");
-      strcat(aux, clover.Twisted() ? ",twist=true" : ",twisted=false");
+      strcat(aux, clover.TwistFlavor() == QUDA_TWIST_SINGLET || clover.TwistFlavor() == QUDA_TWIST_NONDEG_DOUBLET ?
+             ",twist=true" : ",twisted=false");
```

Comment:
Not relevant for correctness, but `twist` vs `twisted` should be reconciled

Reply:
d83e46f5ca2fdc4fec6cba4e396d3ef11e8e6917